test/JDBC/input/BABEL-TABLEOPTIONS-vu-verify.sql

```diff
@@ -1,11 +1,11 @@
-SELECT relname, reloptions FROM pg_class WHERE relname = 'babel_tableoptions_vu_prepare_t1';
```

Comment:
Can you please explain reason behind changing the test cases in this way? because this test case file represent test cases for table options. But with these changes, it will not anymore!

Reply:
I myself added this test case previously and it actually verifies that the upgrade works with all kind of reloptions we introduced in babelfish but it is not necessary to print these reloptions since the output might change everytime support new option.